cms/djangoapps/contentstore/views/component.py

```diff
@@ -220,7 +220,7 @@ def container_handler(request, usage_key_string):
                 'templates': CONTAINER_TEMPATES
             })
     else:
-        return HttpResponseBadRequest("Only supports HTML requests")
+        return HttpResponseBadRequest(_("Only supports HTML requests"))
```

Comment:
This will be a different string than the one at L134 because the capitalization differs. Please match them, using the capital "HTML" string.